test/support/schemas/result_sets/show.json

```diff
@@ -1,7 +1,16 @@
 {
   "properties": {
     "url" : {"type" : "string"},
-    //"identifier" : {"type": "string"},
+    "identifier" : {
+      "type": "array",
+      "items" : {
+        "type" : "object",
+        "properties" : {
+          "system" : {"type" : "string"},
+          "value" : {"type" : "string"}
+        }
+      }
+    },
```

Comment:
I know this isn't really part of the PR, but could you remove the comment line below. Technically, comments aren't allowed in JSON files. I think the Ruby JSON parser does honor them, but others won't.